Formula/ppsspp.rb

```diff
@@ -28,31 +17,64 @@ class Ppsspp < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "nasm" => :build
   depends_on "pkg-config" => :build
-  depends_on "ffmpeg"
-  depends_on "glew"
+  depends_on "python@3.9" => :build
+  depends_on "libpng"
   depends_on "libzip"
   depends_on "sdl2"
   depends_on "snappy"
 
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "molten-vk"
+  end
+
   def install
-    args = std_cmake_args
-    # Use brewed FFmpeg rather than precompiled binaries in the repo
-    args << "-DUSE_SYSTEM_FFMPEG=ON"
+    # Build PPSSPP-bundled ffmpeg from source. Changes in more recent
+    # versions in ffmpeg make it unsuitable for use with PPSSPP, so
+    # upstream ships a modified version of ffmpeg 3.
+    # See https://github.com/Homebrew/homebrew-core/issues/84737.
+    cd "ffmpeg" do
+      rm_rf "macosx/universal"
+      if OS.mac?
+        system "./mac-build.sh"
+      else
+        system "./linux-x86_64.sh"
+      end
+    end
 
-    # fix missing include for zipconf.h
-    ENV.append_to_cflags "-I#{Formula["libzip"].opt_prefix}/lib/libzip/include"
+    # Replace bundled MoltenVK dylib with symlink to Homebrew-managed dylib
+    rm "MoltenVK/macOS/Frameworks/libMoltenVK.dylib"
+    (buildpath/"MoltenVK/macOS/Frameworks").install_symlink Formula["molten-vk"].opt_lib/"libMoltenVK.dylib"
 
     mkdir "build" do
+      args = std_cmake_args + %w[
+        -DUSE_SYSTEM_LIBZIP=ON
+        -DUSE_SYSTEM_SNAPPY=ON
+      ]
+
       system "cmake", "..", *args
       system "make"
+
       if OS.mac?
         prefix.install "PPSSPPSDL.app"
         bin.write_exec_script "#{prefix}/PPSSPPSDL.app/Contents/MacOS/PPSSPPSDL"
         mv "#{bin}/PPSSPPSDL", "#{bin}/ppsspp"
+
+        # Replace app bundles with symlinks to allow dependencies to be updated
+        (prefix/"PPSSPPSDL.app/Contents/Frameworks/libMoltenVK.dylib").unlink
+        (prefix/"PPSSPPSDL.app/Contents/Frameworks/libSDL2-2.0.0.dylib").unlink
+        (prefix/"PPSSPPSDL.app/Contents/Frameworks").install_symlink Formula["molten-vk"].opt_lib/"libMoltenVK.dylib"
+        (prefix/"PPSSPPSDL.app/Contents/Frameworks").install_symlink Formula["sdl2"].opt_lib/"libSDL2-2.0.0.dylib"
```

Comment:
```suggestion
        app_frameworks = prefix/"PPSSPPSDL.app/Contents/Frameworks"
        ln_sf (Formula["molten-vk"].opt_lib/"libMoltenVK.dylib").relative_path_from(app_frameworks), app_frameworks
        ln_sf (Formula["sdl2"].opt_lib/"libSDL2-2.0.0.dylib").relative_path_from(app_frameworks), app_frameworks
```
`install_symlink` will create one that references Cellar paths, and this gets broken on `molten-vk`/`sdl2` version/revision bumps. We don't want that.